astropy/io/fits/card.py

```diff
@@ -859,7 +859,7 @@ def _split(self):
                     return kw, vc
 
                 value = m.group("strg") or ""
-                value = value.rstrip().replace("''", "'")
+                value = value.rstrip()
```

Comment:
First issue, the replacement was done twice, here for the individual CONTINUE cards and a second time when the CONTINUE cards have been joined.

Reply:
Also, for reference, this was refactored in #11108.